pkgs/development/python-modules/skytemple-files/default.nix

```
@@ -21,7 +24,12 @@ buildPythonPackage rec {
 
   buildInputs = [ armips ];
 
-  propagatedBuildInputs = [ appdirs dungeon-eos explorerscript ndspy pillow setuptools skytemple-rust tilequant ];
+  propagatedBuildInputs = [
+    appdirs dungeon-eos explorerscript ndspy pillow setuptools skytemple-rust
+    tilequant pyyaml pmdsky-debug-py
+    # SpriteCollab’s specific dependancies
+    aiohttp lru-dict graphql-core gql
+  ];
```

`optional-dependencies` help to separate the dependencies. Also, `typing-extensions` is [needed](https://github.com/SkyTemple/skytemple-files/blob/1.4.1/setup.py#L48).

```suggestion
  propagatedBuildInputs = [
    appdirs
    dungeon-eos
    explorerscript
    ndspy
    pillow
    pmdsky-debug-py
    pyyaml
    setuptools
    skytemple-rust
    tilequant
  ] ++ lib.optionals (pythonOlder "3.9") [
    typing-extensions
  ];

  passthru.optional-dependencies = {
    spritecollab = [
      aiohttp
      gql
      graphql-core
      lru-dict
    ] ++ gql.optional-dependencies.aiohttp;
  };
```

Depends on https://github.com/NixOS/nixpkgs/pull/200163